web/src/components/Wrapper.js

```diff
@@ -17,7 +18,12 @@ class Wrapper extends Component {
 
   render() {
     const { children } = this.props;
-    return <div className="site">{children}</div>;
+    return (
+      <div className="site">
+        <Header key={location.pathname}/>
```

Comment:
@mgwalker I'm not sure that putting the header in `Wrapper.js` makes sense, but I couldn't figure out where else to put it so that every page would pick it up. I'm open to other ideas!